package.json

```diff
@@ -32,12 +32,14 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
+    "browserify": "^13.1.1",
+    "tsify": "^2.0.7",
     "@types/pixi.js": "^4.3.1",
     "typescript": "^2.1.4"
   },
   "scripts": {
     "demo": "npm run build && open demo/index.html",
-    "build": "tsc",
+    "build": "tsc; browserify pixi-multistyle-text.ts -p tsify -s MultiStyleText -o dist/pixi-multistyle-text.js -d",
```

Comment:
I thought `tsc` was already compiling down to basic JS. Could you explain why browserify is required here?

Also, I'd like to find a way to remove the new wrapped module to remove the duplication`MultiStyleText.MultiStyleText` when the file is injected in a script tag instead of using a commonjs require/import for example.